tcono/src/main/java/blog/econovation/tcono/web/dto/PostUpdateResponseDto.java

```diff
@@ -0,0 +1,35 @@
+package blog.econovation.tcono.web.dto;
+
+import blog.econovation.tcono.domain.post.Post;
+import blog.econovation.tcono.domain.user.User;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+
+import java.time.LocalDateTime;
+
+@Data
+@Getter
+public class PostUpdateResponseDto {
+    private User user;
+    private String content;
+    private String title;
+    private String category; //태그
+    private int views;
+    private int likes;
+    private LocalDateTime createdDate;
+
+    private LocalDateTime modifiedDate;
+
+    @Builder
+    public PostUpdateResponseDto(Post post, String category) {
+        this.user = post.getUser();
+        this.content = post.getContent();
+        this.title = post.getTitle();
+        this.category = category;
+        this.views=post.getViews();
+        this.likes = post.getLike();
+        this.createdDate= post.getCreatedDate();
+        this.modifiedDate=post.getModifiedDate();
```

Comment:
일관된 공백을 사용하는거는 어떨까요?~ 
```suggestion
        this.user = post.getUser();
        this.content = post.getContent();
        this.title = post.getTitle();
        this.category = category;
        this.views = post.getViews();
        this.likes = post.getLike();
        this.createdDate = post.getCreatedDate();
        this.modifiedDate = post.getModifiedDate();
```

Reply:
ctrl + shift + enter : 자동완성 기능을 사용해보세요 인덴트 걱정이 싹~

Reply:
오호 ~ 인텔리제이 기능인가요~ 수민님은 맥북이니까 Command + shift + enter 겠네요 ~